### ISSUE TITLE
Not working on Windows Server 2008R2

### ISSUE THREAD

**Issue by eranvak** [fochess/BypassUAC#1]

**Comment by xsysvermin** [comment]
Ok. Let me try again.

**Comment by rkornmeyer** [comment]
is there any differences with this project compared to https://github.com/hfiref0x/UACME ?

**Comment by MalwareTech** [comment]
Looks like he just ripped all the code from there, can't see any difference between the 2.

**Comment by hfiref0x** [comment]
> xsysvermin commented 2 days ago
> Ok. Let me try again.

You did 3 errors. Correcting: "Ok. Let me **rip** again". Nope.
